apps/meteor/server/models/raw/Messages.ts

```diff
@@ -1310,6 +1320,7 @@ export class MessagesRaw extends BaseRaw<IMessage> implements IMessagesModel {
 
 		if (!limit) {
 			const count = (await this.deleteMany(query)).deletedCount;
+			await ReadReceipts.removeByIdPinnedTimestampLimitAndUsers(rid, pinned, ignoreDiscussion, ts, users, ignoreThreads);
```

Comment:
pls don't call a model from another model

Reply:
Fixed! For this one I had to change the value returned by this method so that we know exactly what messages have been deleted